src/lib/account_update.ts

```diff
@@ -222,6 +222,13 @@ interface Permissions extends Permissions_ {
   // TODO: doccomments
   incrementNonce: Permission;
   setVotingFor: Permission;
+
+  /**
+   * Permission to control the ability to include _any_ account update for this account in a transaction. Note that this is more restrictive than
+   * all other permissions combined. For normal accounts it can safely be set to `none`, but for token contracts this has to be more restrictive, to
+   * prevent unauthorized token interactions -- for example, it could be `proofOrSignature`.
+   */
+  access: Permission;
```

Comment:
Any thoughts on how we default this to non-none somehow when working with tokens? Happy path needs to be the secure one